File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV4001U00ExportCSVInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Invs
{
    public class INV4001U00ExportCSVInfo
    {
        private String _rowNum;
        private String _churiDate;
        private String _importTime;
        private String _jaeryoCode;
        private String _jaeryoName;
        private String _startDate;
        private String _lot;
        private String _expiredDate;
        private String _ipgoQty;
        private String _ipgoDanuiName;
        private String _ipgoType;
        private String _updId;
        private String _ipgoDanga;
        private String _qtyIpgoDanga;
        private String _importCode;
        private String _comment;

        public String RowNum
        {
            get { return this._rowNum; }
            set { this._rowNum = value; }
        }

        public String ChuriDate
        {
            get { return this._churiDate; }
            set { this._churiDate = value; }
        }

        public String ImportTime
        {
            get { return this._importTime; }
            set { this._importTime = value; }
        }

        public String JaeryoCode
        {
            get { return this._jaeryoCode; }
            set { this._jaeryoCode = value; }
        }

        public String JaeryoName
        {
            get { return this._jaeryoName; }
            set { this._jaeryoName = value; }
        }

        public String StartDate
        {
            get { return this._startDate; }
            set { this._startDate = value; }
        }

        public String Lot
        {
            get { return this._lot; }
            set { this._lot = value; }
        }

        public String ExpiredDate
        {
            get { return this._expiredDate; }
            set { this._expiredDate = value; }
        }

        public String IpgoQty
        {
            get { return this._ipgoQty; }
            set { this._ipgoQty = value; }
        }

        public String IpgoDanuiName
        {
            get { return this._ipgoDanuiName; }
            set { this._ipgoDanuiName = value; }
        }

        public String IpgoType
        {
            get { return this._ipgoType; }
            set { this._ipgoType = value; }
        }

        public String UpdId
        {
            get { return this._updId; }
            set { this._updId = value; }
        }

        public String IpgoDanga
        {
            get { return this._ipgoDanga; }
            set { this._ipgoDanga = value; }
        }

        public String QtyIpgoDanga
        {
            get { return this._qtyIpgoDanga; }
            set { this._qtyIpgoDanga = value; }
        }

        public String ImportCode
        {
            get { return this._importCode; }
            set { this._importCode = value; }
        }

        public String Comment
        {
            get { return this._comment; }
            set { this._comment = value; }
        }

        public INV4001U00ExportCSVInfo() { }

        public INV4001U00ExportCSVInfo(String rowNum, String churiDate, String importTime, String jaeryoCode, String jaeryoName, String startDate, String lot, String expiredDate, String ipgoQty, String ipgoDanuiName, String ipgoType, String updId, String ipgoDanga, String qtyIpgoDanga, String importCode, String comment)
        {
            this._rowNum = rowNum;
            this._churiDate = churiDate;
            this._importTime = importTime;
            this._jaeryoCode = jaeryoCode;
            this._jaeryoName = jaeryoName;
            this._startDate = startDate;
            this._lot = lot;
            this._expiredDate = expiredDate;
            this._ipgoQty = ipgoQty;
            this._ipgoDanuiName = ipgoDanuiName;
            this._ipgoType = ipgoType;
            this._updId = updId;
            this._ipgoDanga = ipgoDanga;
            this._qtyIpgoDanga = qtyIpgoDanga;
            this._importCode = importCode;
            this._comment = comment;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV4001U00ExportCSVInfo.java
package nta.med.data.model.ihis.invs;

import java.sql.Timestamp;

public class INV4001U00ExportCSVInfo {
	private String rowNum;
	private Timestamp churiDate;
	private String importTime;
	private String jaeryoCode;
	private String jaeryoName;
	private Timestamp startDate;
	private String lot;
	private Timestamp expiredDate;
	private Double ipgoQty;
	private String ipgoDanuiName;
	private String ipgoType;
	private String updId;
	private Double ipgoDanga;
	private Double qtyIpgoDanga;
	private String importCode;
	private String comment;
	public INV4001U00ExportCSVInfo(String rowNum, Timestamp churiDate, String importTime, String jaeryoCode,
			String jaeryoName, Timestamp startDate, String lot, Timestamp expiredDate, Double ipgoQty,
			String ipgoDanuiName, String ipgoType, String updId, Double ipgoDanga, Double qtyIpgoDanga,
			String importCode, String comment) {
		super();
		this.rowNum = rowNum;
		this.churiDate = churiDate;
		this.importTime = importTime;
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.startDate = startDate;
		this.lot = lot;
		this.expiredDate = expiredDate;
		this.ipgoQty = ipgoQty;
		this.ipgoDanuiName = ipgoDanuiName;
		this.ipgoType = ipgoType;
		this.updId = updId;
		this.ipgoDanga = ipgoDanga;
		this.qtyIpgoDanga = qtyIpgoDanga;
		this.importCode = importCode;
		this.comment = comment;
	}
	public String getRowNum() {
		return rowNum;
	}
	public void setRowNum(String rowNum) {
		this.rowNum = rowNum;
	}
	public Timestamp getChuriDate() {
		return churiDate;
	}
	public void setChuriDate(Timestamp churiDate) {
		this.churiDate = churiDate;
	}
	public String getImportTime() {
		return importTime;
	}
	public void setImportTime(String importTime) {
		this.importTime = importTime;
	}
	public String getJaeryoCode() {
		return jaeryoCode;
	}
	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}
	public String getJaeryoName() {
		return jaeryoName;
	}
	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}
	public Timestamp getStartDate() {
		return startDate;
	}
	public void setStartDate(Timestamp startDate) {
		this.startDate = startDate;
	}
	public String getLot() {
		return lot;
	}
	public void setLot(String lot) {
		this.lot = lot;
	}
	public Timestamp getExpiredDate() {
		return expiredDate;
	}
	public void setExpiredDate(Timestamp expiredDate) {
		this.expiredDate = expiredDate;
	}
	public Double getIpgoQty() {
		return ipgoQty;
	}
	public void setIpgoQty(Double ipgoQty) {
		this.ipgoQty = ipgoQty;
	}
	public String getIpgoDanuiName() {
		return ipgoDanuiName;
	}
	public void setIpgoDanuiName(String ipgoDanuiName) {
		this.ipgoDanuiName = ipgoDanuiName;
	}
	public String getIpgoType() {
		return ipgoType;
	}
	public void setIpgoType(String ipgoType) {
		this.ipgoType = ipgoType;
	}
	public String getUpdId() {
		return updId;
	}
	public void setUpdId(String updId) {
		this.updId = updId;
	}
	public Double getIpgoDanga() {
		return ipgoDanga;
	}
	public void setIpgoDanga(Double ipgoDanga) {
		this.ipgoDanga = ipgoDanga;
	}
	public Double getQtyIpgoDanga() {
		return qtyIpgoDanga;
	}
	public void setQtyIpgoDanga(Double qtyIpgoDanga) {
		this.qtyIpgoDanga = qtyIpgoDanga;
	}
	public String getImportCode() {
		return importCode;
	}
	public void setImportCode(String importCode) {
		this.importCode = importCode;
	}
	public String getComment() {
		return comment;
	}
	public void setComment(String comment) {
		this.comment = comment;
	}
	
}
